package.json

```diff
@@ -29,12 +29,13 @@
     "deep-equal": "1.0.1",
     "invariant": "2.2.0",
     "react-side-effect": "1.0.2",
+    "recompose": "^0.14.5",
```

Comment:
I'm not sure if the tradeoff for a new dependency for the small yet convenient helpers is worth it. It's not that much work anyway to make the HOC for react-helmet.